lib/fastlyctl/commands/logging/bigquery.rb

```diff
@@ -0,0 +1,125 @@
+module BigQuery
+
+    def BigQuery.parse_secret_key(file)
```

Comment:
Might the private keys use a passphrase? Do we need to handle that?

Reply:
TO my knowledge no, Google does not allow you to attach a passphrase (at least from what I have seen)

Reply:
👍